### PR TITLE
MGMT-17586: [IBI] Shutdown the node after the RHCOS image is written to the disk

### PR DESCRIPTION
**Pull request by tsorya**
[MGMT-17586](https://issues.redhat.com//browse/MGMT-17586): [IBI] Shutdown the node after the RHCOS image is written to the disk.

In order to run shutdown command after chroot we need to unchroot adding such option as helper function.

### CODE REVIEW

File: ib-cli/installationiso/data/install-rhcos-and-restore-seed.sh [hunk 1]
```diff
@@ -55,4 +55,8 @@ if [ -n "${PRECACHE_BEST_EFFORT}" ]; then
     additional_flags="${additional_flags} --precache-best-effort"
 fi
 
+if [ -n "${SKIP_SHUTDOWN}" ]; then
+    additional_flags="${additional_flags} --skip-shutdown"
```
[the remark this answers is not in the document]
Comment (tsorya) [reply ib-cli/installationiso/data/install-rhcos-and-restore-seed.sh:59]: The idea was that logic is part of LCA cli and not script , we can though move it to script

**Review by eranco74 — COMMENTED** [review]
/lgtm

**Pull request merged**